supplier_app/tests/test_view.py

```diff
@@ -219,6 +231,18 @@ def test_address_bankaccount_should_be_related_with_taxpayer(self, msg_mocked):
             bankaccount.taxpayer
         )
 
+    @patch('supplier_app.views.messages.add_message')
+    def test_new_taxpayer_should_be_related_with_3_eb_entities(self, msg_mocked):
+        eb_entities = [EBEntityFactory(), EBEntityFactory(), EBEntityFactory()]
```

Comment:
Are we checking we are not having 3 times the same entity?

Reply:
Yes i fixed that